src/spans/awsSpan.js

```diff
@@ -60,6 +62,8 @@ export const getBasicSpan = () => {
   const transactionId = getCurrentTransactionId();
 
   const awsAccountId = getAccountId(lambdaContext);
+  const invokedArn = getInvokedArn();
```

Comment:
What happens if there is an exception thrown from the method, what will happen then? Is it safeguard?

Reply:
It's running inside `safeExecute`